bin/pic-configure

```diff
@@ -46,6 +46,7 @@ help()
     echo "-c | --cmake         - overwrite options for cmake"
     echo "                       (e.g.: \"-DPIC_VERBOSE=21 -DCMAKE_BUILD_TYPE=Debug\")"
     echo "-t <presetNumber>    - configure this preset from cmakeFlags"
+    echo "-f | --force         - flush the cmake cache and scan for new param files"
```

Comment:
Maybe `clear` is more descriptive than `flush` ("cache flush" is actually a different thing)?

Reply:
true, @psychocoderHPC can you please replace `flush` with `clear`?